editor/src/app/user/user-account.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component, OnInit } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { take } from '../../../node_modules/rxjs/operators';
+import { Store } from '@ngxs/store';
+
+@Component({
+  selector: 'berta-user-account',
+  template: `
+    <h2>Change password</h2>
+    <form action="" (submit)="changePassword($event, old_password.value, new_password.value, retype_password.value)">
+      <label for="old_password">Old password
+        <input type="password" name="old_password" #old_password>
+      </label>
+      <label for="new_password">New password
+        <input type="password" name="new_password" #new_password>
+      </label>
+      <label for="retype_password">Retype new password
+        <input type="password" name="retype_password" #retype_password>
+      </label>
+      <button type="submit">Change password</button>
+
+      <div *ngIf="message">
+        {{ message }}
+      </div>
+      <div *ngIf="error">
+        {{ error }}
+      </div>
+    </form>
+
+    <hr />
+    Password must be at least 6 characters long<br>
+    and containing alphanumeric (A-Z, a-z, 0-9) characters.
+  `,
+  styles: [`
+    label {
+      display: block;
+    }
+    input {
+      display: block;
+      margin-bottom: 1rem;
+    }
+    button {
+      margin-bottom: 1rem;
+    }
+  `]
+})
+export class UserAccountComponent implements OnInit {
+
+  error: string;
```

Comment:
You can initialize empty string values here - `error = ''`, `message = ''`.
When you initialize variable with a value you don't have to add type, Because TypeScript can understand what type of value you used.